dont-worry-be-happy/dwbh-api
# Leave group

![](https://media0.giphy.com/media/OAuf5mQLixvOM/giphy.gif?cid=790b76115c9e0eb0696f476859a9e9d2)

# HOW TO TEST
1. Call to leaveGroup on a group where you are a member, it should success
2. Call to leaveGroup on a group where you are not a member, it should fail
3. Call to leaveGroup on a group where you are the unique admin, it should fail
